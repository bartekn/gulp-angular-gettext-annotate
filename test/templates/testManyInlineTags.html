<div>
    <strong>Hello</strong> world <u>underline</u>
</div>