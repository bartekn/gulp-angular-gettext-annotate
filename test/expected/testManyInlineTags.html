<div translate>
    <strong>Hello</strong> world <u>underline</u>
</div>